# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../madwifi/no-find-previous.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- madwifi-hal-0.9.30.10-r2230-20070328/Makefile.vanilla	2007-03-28 21:45:37.694128561 +0200
+++ madwifi-hal-0.9.30.10-r2230-20070328/Makefile	2007-03-28 21:45:52.518973381 +0200
@@ -88,7 +88,7 @@
 install-modules:
 	@# check if there are modules left from an old installation
 	@# might cause make to abort the build
-	sh scripts/find-madwifi-modules.sh $(KERNELRELEASE) $(DESTDIR)
+	#sh scripts/find-madwifi-modules.sh $(KERNELRELEASE) $(DESTDIR)
 
 	for i in $(DIRS_MODULES); do \
 		$(MAKE) -C $$i install || exit 1; \
